.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
   "printWidth": 80,
   "tabWidth": 2,
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none",
+  "arrowParens": "avoid"
```

Comment:
Remove these please. I know the lint results are very noisy right now, but we will be working through soon.